README.md

```diff
@@ -135,7 +135,7 @@ To run as a desktop app:
 
    ```
    npm install electron
-   node_modules/.bin/electron .
+   npm run build:electron
```

Comment:
This section is `To run as a desktop app`
not to build for all platforms (which requires a lot of dependencies, a Mac and luck)